js/bindings/bindings.js

```
@@ -1187,6 +1196,9 @@ class Container {
    * @returns {void}
    */
   allowFrom(srcArg, portRange) {
+    if (portRange === undefined || portRange === null) {
+      return new Error('a port or port range is required');
```

This should be `throw` rather than `return`.